_output.tf

```diff
@@ -53,8 +53,12 @@ output "alb_listener_arn" {
   value = aws_alb_listener.http.arn
 }
 
-output "alb_listener_rules_arns" {
-  value = { for rule_name, rule in aws_alb_listener_rule.this : rule_name => rule.arn }
+output "http_alb_listener_rules_arns" {
```

Comment:
Thinking out loud, it doesn’t feel right that HTTP is the default listener, but since we can’t know in advance whether the user will have a certificate or their domain information to create one for them, this seems to be the only viable option for now.

Reply:
I agree; we can only supply a way, though.